scss/widgets/generic/textEditor/_index.scss

```diff
@@ -650,6 +651,8 @@ $generic-texteditor-label-transition:
       }
     }
 
+    &.dx-dropdowneditor-active,
+    &.dx-dropdowneditor-active.dx-state-hover,
```

Comment:
All elements selected by this selector are also selected by the previous. So it looks useless